tiledb/cf/creator.py

```diff
@@ -646,6 +646,10 @@ def __init__(
         self.offsets_filters = offsets_filters
         self.allows_duplicates = allows_duplicates
         self.sparse = sparse
+        self.__post_init__()
+
+    def __post_init__(self):
+        pass
```

Comment:
curious what this does

Reply:
I wanted to the ability to add extra error checking, etc. to subclasses of the this class. In the base ArrayCreator it does nothing, but it allow subclasses to inherit the `__post_init__` method and use it to do additional things after the object is created, but before it is used.